src/frontend/next/src/components/SearchResults.tsx

```diff
@@ -0,0 +1,104 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { useSWRInfinite } from 'swr';
+import { Container, Box } from '@material-ui/core';
+
+import useSiteMetadata from '../hooks/use-site-metadata';
+import Timeline from './Posts/Timeline';
+import Spinner from './Spinner';
+
+const useStyles = makeStyles(() => ({
```

Comment:
``` js
const useStyles = makeStyles((theme: Theme) => 
createStyles({ spinner: {
    display: 'flex',
    justifyContent: 'center',
  },
  ...
```

Reply:
The theme object is not used here, so it's not needed. It's mostly used to access the palette object, which again, SearchResults is not using here